src/core/effects/qgspainteffect.cpp

```diff
@@ -180,12 +179,7 @@ QRectF QgsPaintEffect::boundingRect( const QRectF &rect, const QgsRenderContext
 
 void QgsPaintEffect::fixQPictureDpi( QPainter *painter ) const
```

Comment:
Can we not deprecated this method and use applyScaleFixForQPictureDpi instead?

Reply:
@troopa81 
Good point, done